frontend/src/routes/index.svelte

```diff
@@ -203,3 +224,11 @@
   currentGenres={$currentGenres}
   {search}
 />
+
+<style>
+  @media (max-width: 768px) {
+    .svelte-select {
+      --margin: 5px;
+    }
+  }
+</style>
```

Comment:
Could this be done with tailwindcss as a `md:m-5`? This would be the only thing in normal CSS, which is a little confusing

Reply:
No

Reply:
:(